controller/config/config.go

```diff
@@ -41,11 +41,11 @@ const (
 	MaxEdgeAPIActivityUpdateInterval      = 10 * time.Minute
 	MinEdgeAPIActivityUpdateInterval      = time.Millisecond
 
-	DefaultEdgeSessionTimeout = 10 * time.Minute
+	DefaultEdgeSessionTimeout = 30 * time.Minute
```

Comment:
add the same comment here please... anything that's used in the config template should get this same comment : `// Don't change the time unit, keep time.Minute`